kubernetes.js

```diff
@@ -727,6 +748,7 @@ module.exports = {
                 (details.body.status?.conditions[0].type === 'Available' ||
                     (details.body.status?.conditions[0].type === 'Progressing' && details.body.status?.conditions[0].reason === 'NewReplicaSetAvailable')
                 )) {
+                    // not calling all endpoints for HA as they should be the same
```

Comment:
This feels a little risky - in case one instance has hit a hang or other unpleasantness that goes unnoticed.

For an initial iteration, we can go with this, but will need to look at next week.